### PR TITLE
data: kaiwalyakoparkar

### PR DESCRIPTION
**Pull request by kaiwalyakoparkar**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

You'll are doing a great job. I would like my LinkFree profile to be updated

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4607"></a>

### CODE REVIEW

**Review by Hrishi5111998 — APPROVED** [review]

**Review by himanshu1221 — DISMISSED** [review]
**LGTM :+1:**
**Thanks For Contributing 🎉**

**Review by stevenovak123 — DISMISSED** [review]

File: data/kaiwalyakoparkar.json [hunk 1]
```diff
@@ -1,18 +1,55 @@
...
       "url": "https://github.com/kaiwalyakoparkar",
       "icon": "FaGithub"
     },
     {
-      "name": "Follow me on Twitter",
-      "url": "https://twitter.com/kaiwalya_",
-      "icon": "FaTwitter"
+      "group": "Support",
+      "name": "GitHub Sponsors: Donate",
+      "url": "https://github.com/sponsors/eddiejaoude",
```
Comment (Panquesito7) [inline data/kaiwalyakoparkar.json:51]: You probably want to update this link.

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: data/kaiwalyakoparkar.json (hunk 1 above)
Comment (kaiwalyakoparkar) [reply data/kaiwalyakoparkar.json:51]: Thank you so much for the review @Panquesito7 I just realised it and came here to fix it :)

**Review by iamthecloverly — APPROVED** [review]

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by kumarsonsoff3 — APPROVED** [review]
Looks Good! 👍

**Pull request merged**